website/views/options/social/index.jade

```diff
@@ -113,3 +113,17 @@ script(type='text/ng-template', id='partials/options.social.html')
   .tab-content
     .tab-pane.active
       div(ui-view)
+
+script(type='text/ng-template', id='modals/copyChatToDo.html')
```

Comment:
Zapping puts the text of the message in the task title field and includes two links in the notes field: a link to the user's member modal and a link to the group where you zapped it. 

The problem is that the notes popover disappears when you move off the task to click on the links, making the links impossible to click.